src/app/public/modules/file-attachment/file-attachment.component.spec.ts

```diff
@@ -0,0 +1,1037 @@
+import {
+  async,
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  Component,
+  DebugElement,
+  ViewChild
+} from '@angular/core';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyFileAttachmentComponent
+} from './file-attachment.component';
+
+import {
+  SkyFileAttachmentsModule
+} from './file-attachments.module';
+
+import {
+  SkyFileAttachmentChange
+} from './file-attachment-change';
+
+import {
+  SkyFileItem
+} from './file-item';
+
+describe('Single file attachment', () => {
+
+  /** Simple test component with tabIndex */
+  @Component({
+    template: `
+      <sky-file-attachment>
+        <div class="sky-custom-drop"></div>
+        <sky-file-attachment-label>
+          Field Label
+        </sky-file-attachment-label>
+      </sky-file-attachment>`
+  })
+  class FileDropContentComponent {
```

Comment:
We should put all component fixtures in the local `fixtures` folder, and create a fixtures module that declares it. For example, https://github.com/blackbaud/skyux-datetime/blob/master/src/app/public/modules/datepicker/fixtures/datepicker.module.fixture.ts.

Reply:
Should I also pull out the fixture from the regular `file-drop` tests, or leave that untouched for now?